content/en/docs/Updating/_index.md

```diff
@@ -6,26 +6,50 @@ description: >
  Keeping the theme up to date.
 ---
 
-We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the theme submodule yourself as follows:
+We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). 
+If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the Docsy theme
+yourself. 
 
-1. In your local copy of your project, run:
+Updating Docsy means that your site will build using the latest version of Docsy at `HEAD` and include 
+all the new commits or changes that have been merged since the point in time that you initially added the Docsy 
+submodule, or last updated. Updating won't affect any modifications that you made in your own project to 
+[override the Docsy look and feel](/docs/adding-content/lookandfeel/), as your overrides 
+don't modify the theme itself. For details about what has changed in the theme, see the list of 
+[Docsy commits](https://github.com/google/docsy/commits/master).
+
+Depending on how you chose to use Docsy, follow the corresponding steps to update the theme:
+
+## Update a Docsy submodule
+
+If you are using the Docsy theme as a submodule in your project, you update the submodule:
+
+1. Navigate to the root of your local project, then run:
 
     ```
     $ git submodule update --remote
     ```
     
-1. Then add and commit your change:
+1. Add and then commit the change to your project:
 
     ```
     $ git add themes/
     $ git commit -m "Updating theme submodule"
     ```
 
-1. Finally, push the change back to the project repo.
+1. Push the commit to your project repo (by default this is your `origin` remote):
```

Comment:
you really don't need to say that this is your origin remote as it's either a)automatically set to origin because you've just made a local copy of your own repo or b)we've explicitly told them to set it to origin if they've copied our example using the command line. It would be extremely weird for it to be anything else.

Reply:
removed
```suggestion
1. Push the commit to your project repo. For example, run:
```